hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -48,7 +48,14 @@
     public static final int ASSERT_TRUE_EVENTUALLY_TIMEOUT;
 
     static {
+        // Activate logging if run from IntelliJ IDEA
+        String libPath = System.getProperty("java.library.path").toLowerCase();
+        if (libPath.contains("idea") || libPath.contains("intellij")) {
```

Comment:
In Turkish alphabet lowercase of capitalized "I" is dotless "ı" instead of "i", so this seems to fail in environments with Turkish locale default. To fix that potential problem, `com.hazelcast.util.StringUtil#LOCALE_INTERNAL` can be passed to method `toLowerCase `.

Btw afaik we have some `eclipse` guys. Searching for intellij may also not work for them.